index.css

```diff
@@ -86,6 +86,7 @@ h3 {
     font-size: 2rem;
     line-height: 3.5rem;
     min-height: 3.5rem;
+    overflow-wrap: break-word;
```

Comment:
This works great but I changed the word-wrap property to the standard overflow-wrap. See this [MDN article](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Text/Wrapping_Text#breaking_long_words) for reference. Thanks!